packages/gatsby-plugin-utils/src/polyfill-remote-file/utils/get-request-headers-for-url.ts

```diff
@@ -0,0 +1,20 @@
+import url from "url"
+
+import type { Headers } from "got"
+import type { Store } from "gatsby"
+
+export function getRequestHeadersForUrl(
+  passedUrl: string,
+  store: Store
+): Headers | undefined {
+  if (!store || !(`getState` in store)) {
+    console.warn(
+      `getRequestHeadersForUrl: argument 2 "store" is not a redux store. Please pass Gatsby's redux store. (for url ${passedUrl})`
+    )
```

Comment:
Let's do soft break for now.
```suggestion
    console.warn(
      `Redux store is required for image CDN please pass Gatsby's redux store. (for url ${passedUrl}). Your site will break in Gatsby V5`
      
      return undefined;
    )
```

Reply:
Sounds good 👍 I'll probably add a check and log it only once too since right now it'll log out for every image

Reply:
Resolved in https://github.com/gatsbyjs/gatsby/pull/35655/commits/bf48099ead677f134be043a21a239a0e821f2d18